site2/docs/developing-binary-protocol.md

```diff
@@ -412,6 +412,12 @@ Parameters:
  * `validation_error` → *(optional)* Indicates that the consumer has discarded
    the messages due to: `UncompressedSizeCorruption`,
    `DecompressionError`, `ChecksumMismatch`, `BatchDeSerializeError`
+ * `properties` -> *(optional)* Reserved configuration items
+ * `txnid_most_bits` -> *(optional)* Same as TC Id, uniquely identify a transaction with `txnid_least_bits`
+ * `txnid_least_bits` -> *(optional)* The id of the transaction opened in a TC, 
+   uniquely identify a transaction with `txnid_most_bits`
+ * `request_id` -> *(optional)* Id for handling response and timeout
```

Comment:
```suggestion
 * `request_id` -> *(optional)* ID for handling response and timeout.
```

Reply:
I also think it may be better to use ID, but other similar places in this file use Id. Is it inappropriate for us to modify the Id here alone?

Reply:
And it feels like Id will look more familiar to engineer

Reply:
We can use "ID" here 
rules: https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.je5thtkj42cb

Reply:
Ok, I will refine this in this file